core/src/main/scala/replpp/scripting/NonForkingScriptRunner.scala

```diff
@@ -43,16 +39,15 @@ object NonForkingScriptRunner {
     // script file, so instead we'll just write it to the beginning of the script file.
     // That's obviously suboptimal, e.g. because it messes with the line numbers.
     // Therefor, we'll display the temp script file name to the user and not delete it, in case the script errors.
-    val predefCode = allPredefCode(config)
-    val scriptCode = Files.readString(scriptFile)
-    val scriptContent = wrapForMainargs(predefCode, scriptCode)
+    val predefCode = "" // TODO drop
+    val scriptContent = wrapForMainargs(predefCode, Files.readString(scriptFile))
```

Comment:
```suggestion
    val scriptContent = wrapForMainargs("", Files.readString(scriptFile))
```

(or is this still used somewhere else I haven't seen?)

Reply:
forgot to cleanup - done now